src/api/util.go

```diff
@@ -12,9 +14,8 @@ type PostReqData struct {
 	Doc    string `json:"doc"`
 }
 
-// 이게 필요한가 ?
-type UserReqData struct {
-	UserID   uint   `json:"user-id"`
+type userUpdateForm struct {
+	ID       uint   `json:"user-id" validate:"required"`
```

Comment:
변수 사용하는거랑 json 형태랑 너무 다르지 않는게 좋을 것 같음. 그리고 json에서는 underline으로 써야할듯
```suggestion
	ID       uint   `json:"id" validate:"required"`
```

혹은

```suggestion
	UserID       uint   `json:"user_id" validate:"required"`
```